setup.py

```diff
@@ -18,7 +18,7 @@
 REQUIREMENTS = [
     "ecl",
     "numpy",
-    "opm>=2020.10.2",  # NB: Pypi versions.
+    "opm==2021.04",  # NB: Pypi versions.
```

Comment:
Looking at https://pypi.org/project/opm/#history, I would guess 2021.04.0 is the correct? Or it might find it still?

Reply:
Hmm, yes seems like you are right. Quite messy version numbering in `opm` it seems. `2021.04` worked though, but I can of course change it 😅
![image](https://user-images.githubusercontent.com/47146384/149747893-c74530cf-e378-4e04-a3e2-7c78c296b313.png)